context/brand-context/default/scss/10-settings/rfs.scss

```diff
@@ -0,0 +1,69 @@
+/**
+ * Responsive Font Sizes v8.0.4
+ * https://github.com/twbs/rfs
+ * https://github.com/twbs/rfs/blob/main/LICENSE
+ */
+
+// Configuration
+// Adapted from:
+// https://github.com/twbs/rfs/blob/v8.0.4/scss.scss
+
+// Base font size
+$rfs-base-font-size: 1.25rem;
+$rfs-font-size-unit: rem;
+
+@if $rfs-font-size-unit != rem and $rfs-font-size-unit != px {
+  @error "`#{$rfs-font-size-unit}` is not a valid unit for $rfs-font-size-unit. Use `px` or `rem`.";
+}
+
+// Breakpoint at where font-size starts decreasing if screen width is smaller
+$rfs-breakpoint: 1200px;
+$rfs-breakpoint-unit: px;
+
+@if $rfs-breakpoint-unit != px and $rfs-breakpoint-unit != em and $rfs-breakpoint-unit != rem {
+  @error "`#{$rfs-breakpoint-unit}` is not a valid unit for $rfs-breakpoint-unit. Use `px`, `em` or `rem`.";
+}
+
+// Resize font size based on screen height and width
+$rfs-two-dimensional: false;
+
+// Factor of decrease
+$rfs-factor: 10;
+
+@if type-of($rfs-factor) != "number" or $rfs-factor <= 1 {
+  @error "`#{$rfs-factor}` is not a valid  $rfs-factor, it must be greater than 1.";
+}
+
+// Generate enable or disable classes. Possibilities: false, "enable" or "disable"
+$rfs-class: false;
+
+// 1 rem = $rfs-rem-value px
+$rfs-rem-value: 16;
```

Comment:
brand context's `typography.scss` currently sets the font-size as `62.5%` on the html element which in effect means `font-size: 10px`  as the baseline font size on our sites. Therefore we set this variable to `10` in our rfs setup on springer to tell the plugin that.

Reply:
https://github.com/twbs/rfs#rem-value-number for reference

Reply:
this is overridden in `springer/scss/10-settings/rfs.scss`

Reply:
if we are going to have 62.5% set as the default for projects using brand context maybe this default should align to prevent every project from having to override it?

Reply:
the settings in `default` are all exactly as `rfs` defined them

Reply:
oh yeah I get that, I'm just saying that we could save everyone the hassle of having to override this by changing this default to `10` which would align with the decision we made in brand context to default the font-size to `62.5%`. Otherwise every project that uses brand-context (which I assume all will?) will need to know that they need to override the `$rfs-rem-value` to `10` in their rfs settings file

Reply:
Yeah, that does make sense. I was taking the approach of exactly replicating `rfs` so the process was the same, but maybe this does make things easier